include/seqan3/search/detail/policy_max_error.hpp

```diff
@@ -91,7 +91,7 @@ struct policy_max_error
 
         [[maybe_unused]] auto query_size = std::ranges::size(query);
 
-        errors.total = to_error_count(total.value, query_size);
+        errors.total = to_error_count(total.error, query_size);
```

Comment:
```suggestion
        errors.total = to_error_count(total.error, query_size);
```

@eseiler, @rrahn  This could now be part of the config now.